### PR TITLE
Add laurette-theatre.fr ruleset

### PR DESCRIPTION
**Pull request by stajan**
Add laurette-theatre.fr ruleset

### CODE REVIEW

File: src/chrome/content/rules/laurette-theatre.fr.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<ruleset name="laurette-theatre.fr">
```
Comment (J0WI) [inline src/chrome/content/rules/laurette-theatre.fr.xml]: There are various subdomains like `[language code].laurette-theatre.fr` that serving a mismatching cert. Could you add a hint about this misbehavior?

File: src/chrome/content/rules/laurette-theatre.fr.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<ruleset name="laurette-theatre.fr">
+        <target host="laurette-theatre.fr" />
```
Comment (J0WI) [inline src/chrome/content/rules/laurette-theatre.fr.xml]: Please use only tabs to indent lines.

**Comment by jeremyn** [comment]
@stajan Are you still interested in working on this?

**Comment by jeremyn** [comment]
I'm closing this pull request due to lack of response from @stajan . I've noted `laurette-theatre.fr` at `https://github.com/EFForg/https-everywhere/issues/3069#issuecomment-353637686` in case someone else wants to make a pull request for this domain.

**Pull request closed without merging**